utils/metrics/add_score.py

```diff
@@ -0,0 +1,18 @@
+import pandas as pd
+
+
+def add_score2csv(score: float, filename: str):
+    try:
+        resultDf = pd.read_csv(filename, index_col=0)
+    except:
+        resultDf = pd.DataFrame(columns=["score"])
+
+    newData = {"score": score}
+    resultDf = resultDf.append(newData, ignore_index=True)
+
+    mean = resultDf.score.mean()
+    std = resultDf.score.std()
+    n = len(resultDf.score)
+
+    print(f"score is {mean} ± {2 * std} (2σ) with n = {n}")
+    resultDf.to_csv(filename, sep=",", index=True)
```

Comment:
looks like here is that `index=True` call if you're interested

Reply:
Not enough more